src/popup/pages/FungibleTokens/TokenDetails.vue

```diff
@@ -154,7 +154,7 @@ export default {
       return this.availableTokens[this.id];
     },
     isAe() {
-      return this.id === AETERNITY_CONTRACT_ID;
+      return this.$route.name === 'coin-details' || this.id === AETERNITY_CONTRACT_ID;
```

Comment:
In my opinion the components should not be aware about the scope around them. The encapsulation should make them blind and the only function for them is to receive some data - be controlled by the scope. So I suggest to add some prop to route meta and check it here instead of comparing the route name.